### DISCUSSION TITLE
Remove Mandatory Custom Attunement Requirements

### DISCUSSION THREAD

**Discussion by NinesCC** [chromiecraft/chromiecraft#6707] [Suggestions]
Attunement requirements for TBC content never existed on the Wrath client, they have been added manually with the mindset that we need to "experience content like it was back then" but this is highly incoherent with all the other changes that have been applied to the server for whatever reasons. Nobody that plays in CC is here to experience TBC as it was back then, because we are playing it with twinked characters on a Wrath client that has different talent trees, spells/abilities and tuning. They are a relic of the past when CC used to try to achieve Blizzlike, but that has been removed from the features of the server on the website/CC Discord a long time ago. Since then, there has been several custom changes like Mythics, Reverred Heroic Reputations removed, nerfs to content difficulty, added a expensive and restrictive Wrath Glyph system, etc.
﻿
Also some of these attunement restrictions are nonsensical and have nothing to do with how things were back in TBC. For example, dungeon attunements like Arcatraz and Shattered Halls were forced for people to be able to queue even for rogues, which if they had lockpicking leveled, never required a key in TBC. The same goes for Karazhan, as for example you don't need to get attuned anymore for it but you still need to do the chain quest for summoning Nightbane. Why do we need all these custom changes?
﻿
If the reason for attunements to exist is testing if the quests work, why are these quests different than any other? People that go for achievements, or want to learn the lore will still complete them regardless. Why are these quests so important?
﻿
If the reason is forcing people to be active, that is not working at all. For example, for the TK attunement, you require a group for a couple of quests, then you need to run 4 Heroics, which have terrible queues times, specially in peak NA times, if you are queuing as DPS this can get up to 90 mins or queue time outs making them impossible to finish and finally you need a raid to complete the questline. If you don't have a guild to support you, getting these done could actually make new players quit, which contradicts some of the server changes lately like nerfing newly released content like SSC because it needs to be more PUG friendly. I will also add that many PUGs struggle getting players because people that could help are not attuned to the latest content on their alts, I've seen this happen in /worldchat.
﻿
One of the reasons people like the Wrath Client is the Quality of Life changes, but they are being removed to add what people hate the most about TBC which are the attunements and reputation grinds. This problem will continue moving forward as we still need to attune for Mount Hyjal and Black Temple, which will have guilds going back to Tier 4 and 5 to get people attuned.

**Comment by Superbolide1** [comment]
Support this idea. Also going to add a suggestion: remove attunement requirement two weeks after raid release. This way players who want to go in on week 1 can still test the attunement quests, and casual players/alts can experience the content starting week 3.

**Comment by Undeadhunter** [comment]
As an alternative, could make the attunement account-wide

**Comment by balleny** [comment]
I see the reasoning behind removing attunement requirements after some time - also earlier than it was back in the days.
Some things have already been treated different from what the progression actually was, including the early releases of inscription DMF cards, engineering goggles etc.

on the other hand chromiecraft wants to progress through the content reasonable closely as to what it was, just on a wotlk client. hence, i see it as a point up for discussion how close or not things should be treated

**Reply by balleny** [reply]
something I would throw into the topic for discussion here is: if we apply the wotlk-like default "early" would be e.g. enabling jewelcrafters missing stuff, and increasing the drop rates of the karazhan enchant recipes to the wotlk 100%

**Reply by NinesCC** [reply]
You make good points. I guess the difference is that attunements are playable content gate keeping and the others are just item gate keeping, the former making you enable to play with others. I think that if you could summon people inside the raids like we used to do on the vanilla brackets the problem would be solved.

**Comment by ishidinho** [comment]
Agree. Please remove them.

"Damn, I love searching Karazhan groups and heroics on a low pop server instead of simply joining my guild's raid tonight." - No one, ever

**Comment by vvranceanu** [comment]
I enjoy being surprised with an attunement requirement. I've never experienced and it's interesting to me. I also don't have 7 alts to complain about that i need to run 3 dungeons with. I think this blown of proportion, and, to be more on the tone of this post, if this was such a bother to you to write all that, consider trying other games as your probably spend too much time in this one, making your patience reach its limits.

There is already a decision of removing the attunement after 2-3 months

**Reply by NinesCC** [reply]
> I've never experienced and it's interesting to me.

You can still do them if you find them interesting the same way you can still do the Karazhan Attunement questline even if it's not required anymore. I am not suggesting for the quests to be removed but the requirement of them to be obligatory. If you want to do them that is fine but others not doing them will have no impact in your experience.

This topic is about content gate keeping by the requirement that all raid members need to finish it for entering an instance. The problem only gets worse once guilds are running Black Temple and they can't get a fresh player to the server into it, because they need to clear SSC to get them attuned so they can play with them.

> I also don't have 7 alts to complain about that i need to run 3 dungeons with.

The TK attunement questline requires: a couple of group quests, 4 heroic dungeons (not 3) and a T4 raid. This was all explained in my post, but I guess you didn't read it.

> I think this blown of proportion, and, to be more on the tone of this post, if this was such a bother to you to write all that consider trying other games as your probably spend too much time in this one, making your patience reach its limits.

I stated facts here, how am I blowing anything out of proportion? It seems you are not very versed in the differences with TBC with a TBC Client, patches releases, changes in said patches, the differences, customization CC is making and what are we playing here, which is Wrath Client locked on level 70 where none of these attunements were required.

I am the leader of one of the most successful guilds on the server and organizing this for 70+ people is quite a lot of unnecessary work, which is the reason I took the time to write this; also, because I can see the problems ahead with future attunements. The main issue is not even the regular players but the fresh ones that just want to catch up... which we, in our low population server, need as many new players as possible.

I do play a lot of different games when I have time, thanks for the suggestion. Since we are trading suggestions, I will suggest you actually read a post before replying to it so you can add something to the conversation.

**Comment by elthehablo** [comment]
The general idea of mod-prog is that we try to mimic what was needed to be done back in the day. The reason we then stray from adding things like MC to Vashj or doing pre-nerf TK is because mechanics wise we want to stay as close as possible to AC. This might seem like cherry picking, and I'm sure there have been examples where this has not been the case, but in general this is the idea we want to maintain.

Let's be honest, if there attunements are not mandatory, hardly anyone will do them. But again, we didn't put them here to pester you, but rather keep the "what-chores-need-to-be-done-before-you-go-kill-a-dragon" idea of vanilla and TBC.

**Reply by NinesCC** [reply]
> The general idea of mod-prog is that we try to mimic what was needed to be done back in the day. 

I don't know, this is very inconsistent line of thinking seeing how things are implemented. Back in the day we had to Min-Max composition for TBC raids, Drums were not raid wide, same as Heroism/Bloodlust or totems were party-wide, many classes didn't have a lot of abilities either, the list goes forever with things that are not being mimiced like back in the day. Can you tell any other example were this logic applies besides these mandatory attunements?

> Let's be honest, if there attunements are not mandatory, hardly anyone will do them. But again, we didn't put them here to pester you, but rather keep the "what-chores-need-to-be-done-before-you-go-kill-a-dragon" idea of vanilla and TBC.

I rather be real, do we have data on how many people are still doing the attunement quests for Karazhan and SSC to this day? Because I see people doing them in my guild every week on their alts even though Kara attunement was removed and they are not planning raiding SSC (for this last week that it remains required), they just do it because they enjoy doing quests. Why is making these quests mandatory so important for the CC project?

In the Vanilla bracket, here in CC back in 2021, you could summon un-attuned people into the raids, these meant that you still wanted people attuned but if someone didn't have it you could work around that and those attunements didn't require a full work day of play time to finish like these in the TBC bracket.

I just don't understand why you need to be so restrictive in a low population server, have you experienced the queues for a DPS role even in peak EU times for a heroic dungeon? let alone less popular times. Some people could be trying to get the quest line finished but will get frustrated with the wait times. Can't you compromise with something in between?

**Comment by heyitsbench** [comment]
I will take this moment to point out the Scrolls of the [Sun](https://wowgaming.altervista.org/aowow/?item=32896) and [Maelstrom](https://wowgaming.altervista.org/aowow/?item=32895), both of which could be used to attune a target for [Tempest Keep](https://wowgaming.altervista.org/aowow/?spell=41601) or [Serpentshrine](https://wowgaming.altervista.org/aowow/?spell=41588) respectively. By my research, these started dropping from Kael/Vashj on live servers back in the day starting with 2.1 until shortly before 2.2 where they were changed to their current version due to attunements no longer being necessary.

**Comment by assassinbait** [comment]
accountwide attuntement sounds like a good compromise

**Comment by shaszun** [comment]
Nines I get your view. BUT. Old players should have completed the attunement when heroics released, like almost an year ago. I had the heroic quests in my quest log for at leas 1-2 months waiting for the dungeons to be unlock. I get the chain is long and kinda of frustrating. If the people of your guild are not helping each other to tackel this, then organize them to do it. In my guild as long people ask we always help them with attunements. But if they prefere RDF and can't complete the attunement before TK release. It's their own fault as being not part of the community in the guild. If this also affects the raids, then they will be having issues with the guild management in some way.

**Reply by heyitsbench** [reply]
What about the guildless?

**Reply by shaszun** [reply]
On a small server to be guildless and TK gear ready it's a lot of red flags. There might be some in Tea's raid, but I would assume it will be a veteran player with big history of joining that pug.

**Reply by NinesCC** [reply]
My guild is very organized and helps all of their players, yet going back to old raids is very taxing on time and puts more strain on management. How do you deal with your new players and how do you fit T4s with SSC and TK atm in your guild's schedule to keep people attuned?

Not everyone is an old player, this whole "forced content" affects the new players the most, which the server needs a lot. Guilds are struggling with numbers, we just had The Lions Den die due to not being able to fill, yet we keep putting this restriction on people who just rolled in CC.

**Comment by NinesCC** [comment]
Can someone from the CC staff please explain why Attunement Quest Lines are held in higher standards than regular quests? Why is it that every other quest in the game is optional but these quests need to be mandatory? They are optional on the WotlK client but are being modified here by the staff to be obligatory.

With the low population of the server and how much rotation there is because of the problems with raids, why is this gatekeeping necessary? Why are these quests more important than other quests?

**Comment by Zelfie** [comment]
Given the servers population, surely gatekeeping anything with attunements, in any way, for any time, is adding an unnecessary barrier for entry that the server could do without.

**Comment by BeefSteaker** [comment]
Closing as the approach to attunements is being changed. per community vote, attinements will only be required for the first month of release
